### ISSUE TITLE
FrameTables vs. VesselGrid

### ISSUE THREAD

**Issue by zooid** [OCXStandard/OCX_Schema#30]
Hi,

Does it make sense to others as well that these two elements can be merged into one?
It appears to me that they are complementing each other but generally defining the same thing.

![image](https://github.com/OCXStandard/OCX_Schema/assets/1837230/33886322-5feb-4430-b784-4cfd6ce88a8a)

![image](https://github.com/OCXStandard/OCX_Schema/assets/1837230/f5306c9f-73b9-450d-a0b7-3c0979559d6a)

I know there may be resistance but could the name "FrameTables" be changed to "PlaneTables"? "Frame" is widely identified by transverse (XY) but here we also have ZX and YZ planes?

**Comment by ocastrup** [comment]
We admit the naming is confusing, but this is due to historic reasons and backward compatibility.
ocx:FrameTables came first and was used to define the reference planes. The ref planes are used as limits (with an optional offset) in order to define the Panel limits in LimitedBy. 
Later, the need came to define the naval architect frame table coordinate system. The FrameTables name we had already used, so instead of breaking the backward compatibility we invented the VesselGrid.
The two serve different purposes, the VesselGrid is a parametric definition of the coordinate system. It can be transferred to a receiving application to be used for further modelling. VesselGrid shall not be used as a LimetdBy limit.

What we can discuss, is the renaming of the FrameTables to e.g. ReferencePlanes and VesselGrid to VesselFrameTable
